config/base_config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "apiKey": "AIzaSyAc19CssqMyByeYlh4XjkH9gKNaM4Q0N2E",
+  "authDomain": "nook-emergency-0-reporting.firebaseapp.com",
+  "databaseURL": "https://nook-emergency-0-reporting.firebaseio.com",
+  "projectId": "nook-emergency-0-reporting",
+  "storageBucket": "nook-emergency-0-reporting.appspot.com",
+  "messagingSenderId": "742293946445",
+  "appId": "1:742293946445:web:5bf9951be93e22ec43dc04",
+  "measurementId": "G-S4J6TZ44LN",
+  "mapboxKey": "pk.eyJ1IjoiZWxheWFiaGFyYXRoIiwiYSI6ImNqMTRra3JubDAwMDYzM25uOXEwNDF3ZzIifQ.ntzLwhDXekJpMtyMpygz7g",
+  "mapboxStyleURL": "mapbox://styles/elayabharath/cjou0w3qa91ww2rp6gcafy3pt"
+}
```

Comment:
would it be worth describing in the readme what the base_config.json and the project_config.json files are (and what the difference between them is)?

Reply:
Updated the readme with a description of both of them